core/modules/admin_bar/admin_bar.theme.inc

```diff
@@ -115,3 +124,11 @@ function template_preprocess_admin_bar_icon(&$variables) {
   $variables['alt'] = t('Home');
 }
 
+/**
+ * Preprocesses variables for theme_admin_bar_alert_icon().
+ */
+function template_preprocess_admin_bar_alert_icon(&$variables) {
+  $variables['src'] = $GLOBALS['base_url'] . '/' . backdrop_get_path('module', 'admin_bar') . '/images/alert-icon.png';
```

Comment:
I'm not getting the alert-icon.png. Did you create one and maybe it's not attached to this PR?